code/controllers/configuration/entries/game_options.dm

```diff
@@ -66,8 +66,6 @@
 
 /datum/config_entry/flag/disable_peaceborg
 
-/datum/config_entry/flag/disable_warops
```

Comment:
wait, if we already have a config option for this, doesn't that mean that it's the admin team's decision as to whether or not to disable war ops, not ours?

Reply:
like, if you wanna remove war ops, just ask the headmins to hold a vote for turning off this config option on our servers. there's no need to remove war ops from the code entirely.

Reply:
The config has been removed. Rise up against admemes.